src/test/java/org/traccar/protocol/Vt200ProtocolDecoderTest.java

```diff
@@ -43,9 +44,6 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, binary(
                 "28631037309456208200210103302307171805444417097301147188170198090f0000073a002000007e00074429"));
 
-        verifyNull(decoder, binary(
```

Comment:
Why are we removing this?

Reply:
That was used to confirm that Traccar returns null when this type of data is received. It was there because the decoding of this type of data was not implemented but now we are decoding so it won’t return null anymore. The type of data is defined by the 3094 in the 28631037**3094**56208200210103302307171805444417097301147188170198090f0000073a002000007e00074429

Reply:
Still doesn't explain why we need to remove it instead of just changing it.

Reply:
Yes. I would prefer not to remove existing tests.

Reply:
Sure. Would that be the only required change?

Reply:
As I said, I need documentation to review the rest.

Reply:
Just attach to the ticket or provide a link.

Reply:
[OBD VT200(400) Protocol_V3.1.1.docx](https://github.com/traccar/traccar/files/6238508/OBD.VT200.400.Protocol_V3.1.1.docx)